### ISSUE TITLE
Unable to Use Nuget to Install OsmApiClient

### ISSUE THREAD

**Issue by stephbuon** [blackboxlogic/OsmApiClient#35]
Hi, I am trying to use Nuget PM to install OsmApiClient. However, even after I try `NuGet\Install-Package OsmApiClient -Version 0.0.8`, OsmApiClient is unrecognized. 

I get this error: **The type or namespace 'OsmApiClient' could not be found (are you missing a using directive or assembly reference?)
I am calling the assembly like so: `using OsmApiClient;`**

The installation does not throw any errors: 

**PM> NuGet\Install-Package OsmApiClient -Version 0.0.8
Restoring packages for C:\Users\steph\source\repos\unity_map_test\unity_map_test\unity_map_test.csproj...
Installing NuGet package OsmApiClient 0.0.8.
Assets file has not changed. Skipping assets file writing. Path: C:\Users\steph\source\repos\unity_map_test\unity_map_test\obj\project.assets.json
Restored C:\Users\steph\source\repos\unity_map_test\unity_map_test\unity_map_test.csproj (in 18 ms).
Executing nuget actions took 34 ms
Time Elapsed: 00:00:00.0904549**

I am using Visual Studio 2022. 

Advice would be greatly appreciated! Thank you for your time and help!

**Comment by HarelM** [comment]
The package works, I use it in production.
I have no clue what your setup is, but you need to make sure you are using dotnet core or latest releases of .net...

**Comment by blackboxlogic** [comment]
@stephbuon 

Could you try version 0.0.7 and comment here if it is the same result.

Try checking out @HarelM 's project and see if that builds on your computer. I don't know if this project is complicated and could fail for other reasons https://github.com/IsraelHikingMap/Site

Is your project in a public repo that I could check out to test? (I didn't see a "unity map test" project in your GitHub).

Try the normal dance visual studio sometimes needs: clean/rebuild/close/open.

**Comment by stephbuon** [comment]
Thank you to both of you for your advice. I am using Visual Studio 2022 and am using .NET 6.0 for long term support. Should I try .NET 7.0 instead?

I tried OsmApiClient 0.0.7 and have the same result. 

I tried: Build/Clean Solution and Build/Rebuild Solution but still have the same result. 

I put the code on a public repository. If you have the time, I would be very grateful for your opinion on why there might be a problem. 

Repository: https://github.com/stephbuon/unity-mapping

C# code that uses OsmApiClient: https://github.com/stephbuon/unity-mapping/blob/main/unity_map_test/Program.cs (You will see there's not much there) 

Thanks again!
